web/docs/integrations/google.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Google Integrations
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Google Integrations
+
+## Google Auth
+
+To use Google as an authentication method (covered [here](/docs/language/features#google)), you'll first need to create a Google project and provide Wasp with your client key and secret. Here is how to do so:
+
+- Create a Google Cloud Platform account if you do not already have one: https://cloud.google.com/
+- Create and configure a new Google project here: https://console.cloud.google.com/home/dashboard
+  - ![Google Console Screenshot 1](../../static/img/integrations-google-1.jpg)
+  - ![Google Console Screenshot 2](../../static/img/integrations-google-2.jpg)
+- Search for `OAuth` in the top bar, click on `OAuth consent screen`
+  - ![Google Console Screenshot 3](../../static/img/integrations-google-3.jpg)
+  - Select what type of app you want, we will go External
+    - ![Google Console Screenshot 4](../../static/img/integrations-google-4.jpg)
+  - Fill out applicable information on Page 1
+    - ![Google Console Screenshot 5](../../static/img/integrations-google-5.jpg)
+  - On Page 2, Scopes, search for `email`
+    - Select the `userinfo.email` row
+      - ![Google Console Screenshot 6](../../static/img/integrations-google-6.jpg)
+      - ![Google Console Screenshot 7](../../static/img/integrations-google-7.jpg)
+      - ![Google Console Screenshot 8](../../static/img/integrations-google-8.jpg)
+  - Add any test users you want on Page 3
+    - ![Google Console Screenshot 9](../../static/img/integrations-google-9.jpg)
+- Next, click `Credentials`
+  - ![Google Console Screenshot 10](../../static/img/integrations-google-10.jpg)
+  - Select `+ Create Credentials`
+  - Select `OAuth client ID`
+    - ![Google Console Screenshot 11](../../static/img/integrations-google-11.jpg)
+  - Complete the form
+    - ![Google Console Screenshot 12](../../static/img/integrations-google-12.jpg)
+  - Under Authorized URIs, put in: `http://localhost:3000/auth/redirect/google`
+    - ![Google Console Screenshot 13](../../static/img/integrations-google-13.jpg)
+    - You can optionally put in any Heroku or other PaaS URL where your API server will be deployed
```

Comment:
"optionally" -> it is required however, if they want for it to be useable in production, right? So maybe we can phrase it as `Once you know on which URLs your API server will be deployed, also add those URLs`, or smth like that?

Reply:
Good point, I will use phrasing similar to yours to clarify that, thanks.